staker-rewards/lib/fraud-detector.js

```diff
@@ -0,0 +1,42 @@
+import clc from "cli-color"
+
+export default class FraudDetector {
+  constructor(context, tokenStaking, factoryDeploymentBlock) {
+    this.context = context
+
+    this.tokenStaking = tokenStaking
+
+    this.factoryDeploymentBlock = factoryDeploymentBlock
+  }
+
+  static async initialize(context) {
+    const { TokenStaking, factoryDeploymentBlock } = context.contracts
+
+    const tokenStaking = await TokenStaking.deployed()
+
+    return new FraudDetector(context, tokenStaking, factoryDeploymentBlock)
+  }
+
+  async isOperatorFraudulent(operator) {
+    console.log(`Checking fraudulent activity for operator: ${operator}`)
```

Comment:
We do not put colon for the rest of the logs:
https://github.com/keep-network/keep-ecdsa/blob/78ab836fa005415a47a4388ff2e19b70e750d6c0/staker-rewards/lib/sla-calculator.js#L82

Reply:
Yeah, but maybe we should? I'll remove it for consistency...

Reply:
(Also, linter is unhappy.)